cli/options/options.go

```diff
@@ -205,7 +211,43 @@ func GetConfigFromContext(ctx *cli.Context) (config.Config, error) {
 	if argCp := ctx.String("config-path"); argCp != "" {
 		configPath = argCp
 	}
-	return config.Load(configPath, GetNetwork(ctx), relativePath)
+
+	network := GetNetwork(ctx)
+	switch network {
+	case netmode.MainNet:
+		return getEmbeddedConfig(configFiles.MainnetConfig)
+	case netmode.TestNet:
+		return getEmbeddedConfig(configFiles.TestnetConfig)
+	case netmode.UnitTestNet:
+		return getEmbeddedConfig(configFiles.UnitTestnetConfig)
+	case netmode.PrivNet:
+		return getEmbeddedConfig(configFiles.PrivnetConfig)
```

Comment:
This should be placed inside `func LoadFile(configPath string, relativePath ...string) (Config, error) {`. There you have a `configData, err := os.ReadFile(configPath)`, so this `configData` should be taken either from embedded files or from the real file depending on the `configPath`.